Keas.Mvc/Keas.Mvc.csproj

```diff
@@ -31,6 +31,7 @@
   </ItemGroup>
   <ItemGroup>
     <Folder Include="Data\" />
+    <Folder Include="Pages\" />
```

Comment:
why?

Reply:
Oops, didn't see that. It is where the view got created. No idea why.